src/cmds/qstat.c

```diff
@@ -294,6 +309,47 @@ states(char *string, char *q, char *r, char *h, char *w, char *t, char *e, int l
 	}
 }
 
+/**
+ * @brief
+ *	Prepend escape character with special characters such as line feed and form feed
+ *
+ * @param[in]   val - string which needs to be formatted
+ *
+ * @return string
+ * @retval NULL     - if memory allocation failed.
+ * @retval string   - formatted string.
+ *
+ */
+char *
+escape_feed_chars(char *val) {
```

Comment:
corrected my comment - we are replacing a '\n' with a slash '\\' and a 'n'. My thinking is it is called escaping when we add a '\\' before '\n'.
coming to the name - if not encode - i can think of convert_feed_chars..